web-ui/src/components/notes/Note.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useContext, useEffect, useRef, useState } from "react";
+import { getNoteByCheckinId, updateCheckinNote } from "../../api/checkins";
+import useDebounce from "../../hooks/useDebounce";
+import NotesIcon from "@material-ui/icons/Notes";
+import LockIcon from "@material-ui/icons/Lock";
+import { AppContext } from "../../context/AppContext";
+
+import "./Note.css";
+
+const Notes = (props) => {
+  const { state } = useContext(AppContext);
+  const { userProfile } = state;
+  const canViewPrivateNote =
+    userProfile.role.includes("PDL") || userProfile.role.includes("ADMIN");
+  const { checkin, memberName } = props;
+  const { id } = checkin;
+  const [note, setNote] = useState({});
+  // TODO: get private note and determine if user is PDL
```

Comment:
TODO?

Reply:
Yeah that will have to be a different story. As far as I'm aware, there isn't a way to get a private note yet

Reply:
@mkimberlin Is there going to be a private note endpoint?

Reply:
Earlier it was suppose to be part of checkin note service but removed it from there. I believe there will be a different service to handle private notes.